app/models/host_aggregate.rb

```diff
@@ -0,0 +1,43 @@
+class HostAggregate < ApplicationRecord
+  include NewWithTypeStiMixin
+  include Metric::CiMixin
+  include EventMixin
+  include ProviderObjectMixin
+  include VirtualTotalMixin
+
+  acts_as_miq_taggable
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
+
+  # FIXME: I don't think this is needed here
```

Comment:
@Loicavenel We've got  this in the model already in terms of host associations, etc. (look at the rest of the files in this PR and you'll see that) I said that a host aggregate *can* map 1-to-1 with an availability zone (i.e. if it's created with an availability_zone attribute), but of course it doesn't have to map to one. I was only speaking in the context of rollups -- i.e. if Host Aggregates will sometimes correspond to availability zones, the same sort of rollups that we do for availability zones would make sense for aggregates.

As for the provisioning comment, the user actually won't choose a host aggregate directly. For host aggregates that are linked to availability zones, choosing an AZ will choose an aggregate. For others, the matching is behind the scenes -- i.e. a user chooses a flavor with certain metadata attributes, and the host aggregates are also defined with metadata attributes, so nova will match those up.